reports/final/report.tex

```diff
@@ -57,16 +57,16 @@
 \pagenumbering{gobble}
 \section*{Executive Summary}
 % ERIC OWNS THIS SECTION
-The Covid-19 pandemic abruptly forced many educational institutions towards online
-learning. As a result, many of these institutions were unequipped to deal with this
-new learning model due to, for example, a lack of laboratory equipment to provide
-to students. Much of the onsite equipment is ill-suited for online learning due to
-its high cost and poor portability. Additionally, the take-home equipment provided
-to students during the pandemic was typically of lower quality in comparison. In
-particular, the oscilloscopes provided by the Electronics II course taught at Queen's
-University were expensive and of poor quality. They had tiny 3-inch displays and a
-small maximum bandwidth of 100 kHz. Moreover, they were single channel, limiting the
-student's ability to see how signals in a circuit are related to one another.
+The Covid-19 pandemic abruptly forced educational institutions towards online
+learning. As a result, many of these institutions were unequipped to deal with
+this new learning model. Much of the onsite equipment is ill-suited for online
```

Comment:
Would be better to at least mention labs, it is unclear what type of "equipment" is referred to here
E.g. Much of the onsite lab equipment ...

Reply:
I could be more specific:

> Much of the onsite equipment, such as oscilloscopes and signal generators, are ill-suited for online...

Reply:
Sure

Reply:
4ea7142ff22429b6e97ec58dfaae83ca7a177618